solver/src/http_solver.rs

```diff
@@ -424,6 +437,7 @@ mod tests {
             mock_token_info_fetcher,
             mock_price_estimation,
             "mock_network_id".to_string(),
+            1.,
```

Comment:
Would this also be a place to use the default instead of a hard coded 1?

Reply:
You mean that `Default::default()` thing? I assumed that that is just a trait that can get specialized for every type, and for f64 it is zero. And currently zero is the only value forbidden for `fee_discount_factor`  (see previous comment).